test/deploy.js

```diff
@@ -91,3 +92,23 @@ test.serial('deployAll --prune', (t) => {
     }))
     .then(() => deploy.deployAll({ prune: true }));
 });
+
+test.serial('deployAll with .DS_Store files', (t) => {
```

Comment:
If I revert actual fix, then the promise is rejected, and the ava framework considers this a failure. With the fix, it's a success. I can add more assertions if you like, but I'm working with the implied promise assertion so far.